.github/workflows/link-checker.yml

```diff
@@ -37,6 +40,9 @@ jobs:
             https://raw.githubusercontent.com/balena-os/meta-balena
             https://github.com/myorg/myapp
             https://index.docker.io/v2/
+            http://api.resin.io/
+            https://api.balena-cloud.com/
+            https://hub.docker.com/r/balenalib/
```

Comment:
I was thinking to find an option where we can ignore any URL that goes 429 while checking. I don't think putting hub.docker.com might be the best solution. It's at least checking for several thousand URL's successfully and makes sure base images are actually present.